c2corg_ui/static/js/map/map.js

```diff
@@ -166,10 +166,14 @@ app.MapController = function($scope, mapFeatureCollection) {
       this.map.addInteraction(clickInteraction);
     }
   } else {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: this.zoom_ || app.MapController.DEFAULT_ZOOM
-    }));
+    // no special feature displayed on the map => use the default extent
+    this.map.on('postrender', function(event) {
```

Comment:
It feels wrong to create a map without center. Couldn't you use the center of the extent?